packages/database/src/aggregated-l2-cost/repository.ts

```diff
@@ -20,32 +19,31 @@ export class AggregatedL2CostRepository {
   async addOrUpdateMany(records: AggregatedL2CostRecord[]): Promise<number> {
     const rows = records.map(toRow)
 
-    await this.db.transaction().execute(async (trx) => {
-      for (let i = 0; i < rows.length; i += BATCH_SIZE) {
-        await trx
-          .insertInto('public.aggregated_l2_costs')
-          .values(rows.slice(i, i + BATCH_SIZE))
-          .onConflict((cb) =>
-            cb.columns(['timestamp', 'project_id']).doUpdateSet({
-              total_gas: (eb) => eb.ref('excluded.total_gas'),
-              total_gas_eth: (eb) => eb.ref('excluded.total_gas_eth'),
-              total_gas_usd: (eb) => eb.ref('excluded.total_gas_usd'),
-              blobs_gas: (eb) => eb.ref('excluded.blobs_gas'),
-              blobs_gas_eth: (eb) => eb.ref('excluded.blobs_gas_eth'),
-              blobs_gas_usd: (eb) => eb.ref('excluded.blobs_gas_usd'),
-              calldata_gas: (eb) => eb.ref('excluded.calldata_gas'),
-              calldata_gas_eth: (eb) => eb.ref('excluded.calldata_gas_eth'),
-              calldata_gas_usd: (eb) => eb.ref('excluded.calldata_gas_usd'),
-              compute_gas: (eb) => eb.ref('excluded.compute_gas'),
-              compute_gas_eth: (eb) => eb.ref('excluded.compute_gas_eth'),
-              compute_gas_usd: (eb) => eb.ref('excluded.compute_gas_usd'),
-              overhead_gas: (eb) => eb.ref('excluded.overhead_gas'),
-              overhead_gas_eth: (eb) => eb.ref('excluded.overhead_gas_eth'),
-              overhead_gas_usd: (eb) => eb.ref('excluded.overhead_gas_usd'),
-            }),
-          )
-          .execute()
-      }
+    // NOTE: This used to be 10_000, but was changed. (sz-piotr): No idea why
```

Comment:
I guess it has something to do with the prepared query param count limit? cc: @sdlyy

Reply:
yessirr